testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeDeltaLakeOss.java

```diff
@@ -60,7 +60,7 @@
 
     private static final String SPARK_CONTAINER_NAME = "spark";
 
-    private static final String DEFAULT_S3_BUCKET_NAME = "trino-ci-test";
+    private static final String S3_BUCKET_NAME = "test-bucket";
```

Comment:
Just for context, `EnvSinglenodeHudi` uses the same strategy as you did here:

https://github.com/trinodb/trino/blob/0ef6960cc7b4773527d8c05899ac1f78af79d158/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeHudi.java

Reply:
-> https://github.com/trinodb/trino/pull/17981